test/configure.in

```diff
@@ -15,7 +15,7 @@ AC_C_CONST
 
 dnl Figure out which math library to use
 case "$host" in
-    *-*-cygwin* | *-*-mingw32*)
+    *-*-cygwin* | *-*-mingw32* | *-windows*)
```

Comment:
I did not update ./configure after changing these due to a lack of the correct autoconf version.

This must be fixed.